JuiceMaker/JuiceMaker/Model/JuiceMaker.swift

```diff
@@ -1,28 +1,35 @@
 //
 //  JuiceMaker - JuiceMaker.swift
-//  Created by yagom. 
+//  Created by yagom.
 //  Copyright © yagom academy. All rights reserved.
 //  last modified by Yetti, yy-ss99, Mary.
 
 struct JuiceMaker {
-    let fruitStore: FruitStore
+    let fruitStore =  FruitStore()
```

Comment:
띄어쓰기가 두번 들어가있네요..! 😅
```suggestion
    let fruitStore = FruitStore()
```

Reply:
해당 부분 수정하였습니다! 😂